build/presubmit.yaml

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Terraform version used is 0.14.7.
+# Terraform version used is 0.14.8.
+substitutions:
+  _CFT_TOOLS_CONTAINER: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:8b0d85766044b0e4701729f9e70ee78dcecdb8688bd86915eb2a1cb00580ba1e"
```

Comment:
does the light image work for our own CICD as well?

Reply:
No, it's missing `go`.